.eslintrc.js

```diff
@@ -127,6 +129,10 @@ module.exports = {
       },
       parser: '@typescript-eslint/parser',
     },
+    {
+      files: ['**/tsconfig*.json', '**/*.json5', '**/*.jsonc'],
```

Comment:
```suggestion
      files: ['**/*.tsconfig.json', '**/*.json5', '**/*.jsonc'],
```

`whatever.tsconfig.json` is more common naming pattern for split out tsconfigs, wdyt?

Reply:
ehh, we already have files called `tsconfig.<foo>.json`, so this would necessitate renaming them